plugins/logitech-scribe/fu-logitech-scribe-device.c

```diff
@@ -561,6 +562,9 @@ fu_logitech_scribe_device_write_firmware(FuDevice *device,
 	 * image file pushed. Device validates and uploads new image on inactive partition. Reboots
 	 * wait for RemoveDelay duration
 	 */
+	fu_device_sleep_full(FU_DEVICE(self), 60 * 1000, fu_progress_get_child(progress));
```

Comment:
I'm confused what this sleep is achieving? Doesn't it work without this? All this is doing is blocking the mainloop so that the uevent is processed slightly later.

Reply:
Sleep is needed. It doesn't solve the issue without Sleep. By the time code hit Sleep, device is already gone/disconnected/rebooting.

Reply:
Okay, I'm really confused why the sleep would work here -- is the device somehow getting confused when we probe it as soon as it enumerates as a new device? e.g. I'd be interested in the difference between `sudo fwupdtool --plugins logitech_scribe install-blob THE_PAYLOAD -vv --no-timestamp` between the sleep and no-sleep variants. i.e. this smells like a workaround that we need to investigate a bit more.